### PR TITLE
Update autogluon tests to resolve pip check issue

### PR DESCRIPTION
**Pull request by arjkesh**
*GitHub Issue #, if available:*

Note: If merging this PR should also close the associated Issue, please also add that Issue # to the Linked Issues section on the right. 

### Description
- pip check failure regarding pytorch lightning version
- update version pins to resolve

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license. I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: autogluon/inference/docker/0.4/py3/Dockerfile.cpu [hunk 1]
```diff
@@ -30,7 +30,7 @@ RUN pip --no-cache-dir install --upgrade --trusted-host pypi.org --trusted-host
 
 # Security patching
 RUN pip install --no-cache-dir -U 'ray>=1.10,<1.11' \
- && pip install --no-cache-dir -U 'pytorch_lightning>=1.5.10,<1.7.0' \
+ && pip install --no-cache-dir -U 'pytorch_lightning>=1.5.10,<1.6.0' \
```
Comment (gradientsky) [inline autogluon/inference/docker/0.4/py3/Dockerfile.cpu:33]: I suggest pinning setuptools version instead where setuptools is installed
```shell
pip install setuptools==59.5.0
```

Comment (arjkesh) [reply autogluon/inference/docker/0.4/py3/Dockerfile.cpu:33]: will that pin version of pytorch lightning as well?

Comment (gradientsky) [reply autogluon/inference/docker/0.4/py3/Dockerfile.cpu:33]: Please disregard the comment, I thought it was related to different issue.

**Review by gradientsky — DISMISSED** [review]

**Review by gradientsky — APPROVED** [review]

**Review by kevinyang8 — APPROVED** [review]

**Pull request merged**